src/Selenium2Library/keywords/_logging.py

```diff
@@ -12,7 +12,17 @@ def _debug(self, message):
         logger.debug(message)
 
     def _get_log_dir(self):
-        variables = BuiltIn().get_variables()
+        # Use screenshot root directory if set
+        if self.screenshot_root_directory is not None:
+            return self.screenshot_root_directory
+
+        # If robotframework isn't running use current directory
+        try:
+            variables = BuiltIn().get_variables()
+        except RobotNotRunningError:
+            return os.getcwd()
```

Comment:
1. `RobotNotRunningError` doesn't seem to be imported.
2. Why not just return `.` instead of `os.getcwd`? In most cases the end result is the same.
3. I was thinking  `screenshot_root_directory` would be returned only if Robot isn't running, but always returning it if defined may be a useful feature in general.